source/initial_temperature/adiabatic_boundary.cc

```diff
@@ -20,65 +20,45 @@
 
 
 #include <aspect/initial_temperature/adiabatic_boundary.h>
-#include <fstream>
-#include <iostream>
+#include <aspect/geometry_model/interface.h>
+#include <aspect/utilities.h>
+#include <deal.II/base/signaling_nan.h>
 
 namespace aspect
 {
   namespace InitialTemperature
   {
-    template <>
-    double
-    AdiabaticBoundary<2>::get_isotherm_depth(const double,
-                                             const double) const
-    {
-      AssertThrow (false, ExcMessage ("The `adiabatic boundary' initial temperature plugin is only implemented for 3d cases."));
-      return 0;
-    }
-
-    template <>
-    double
-    AdiabaticBoundary<3>::get_isotherm_depth(const double latitude,
-                                             const double longitude) const
-    {
-      /**
-       * Loop over the entire array and see if we find a point
-       * that's within delta of what we're looking for. If this
-       * point is not within the delta then test if it is within twice
-       * as much as delta.
-       */
-      for (unsigned int i = 0; i <= depths_iso.size();)
-        {
-          if (std::fabs(latitude - latitudes_iso[i]) <= delta && std::fabs(longitude - longitudes_iso[i]) <= delta)
-            {
-              return depths_iso[i];
-            }
-          else
-            i++;
-        }
-      Assert (false, ExcInternalError());
-      return 0;
-    }
+    template <int dim>
+    AdiabaticBoundary<dim>::AdiabaticBoundary ()
+      :
+      surface_boundary_id(1)
```

Comment:
Please make this an invalid unsigned int to make sure it will throw an error if somebody tries to use it, before it is assigned.
```suggestion
      surface_boundary_id(numbers::invalid_unsigned_int)
```